src/rpc/misc.cpp

```diff
@@ -1063,8 +1063,8 @@ UniValue getaddresstxids(const UniValue& params, bool fHelp)
 UniValue getspentinfo(const UniValue& params, bool fHelp)
 {
     std::string disabledMsg = "";
-    if (!fExperimentalInsightExplorer) {
-        disabledMsg = experimentalDisabledHelpMsg("getspentinfo", "insightexplorer");
+    if (!(fExperimentalInsightExplorer || fExperimentalLightWalletd)) {
+        disabledMsg = experimentalDisabledHelpMsg("getspentinfo", {"insightexplorer", "lightwalletd"});
```

Comment:
Revert these changes; `getspentinfo` cannot be called with `-lightwalletd` because this RPC requires `fSpentIndex` which `-lightwalletd` does not enable.

Reply:
Right, fixed at https://github.com/zcash/zcash/pull/4402/commits/f65acd0aeea44cc4d3ab058ed8b870fb3ba5b595